### PR TITLE
Chapter3

### PR DESCRIPTION
**Pull request by Ruslan357**
Tasks from chapter 3

### CODE REVIEW

File: 3.14.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+                #Exercise 1 
+
+def right_justify(s):
+    while len(s)!=70:
+        s=' '+s
```
Comment (auraz) [inline 3.14.py]: there is neatpick in python, that allow to do `s = ' '*69 + s`

File: 3.14.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    square_part1()
+    for i in range(4):
+        square_part2()
+    square_part3()
+    for i in range(4):
+        square_part2()
+    square_part1()
+   
+square()
+
```
Comment (auraz) [inline 3.14.py:96]: please do four

**Review by auraz — CHANGES_REQUESTED** [review]

File: 3.14.py (hunk 2 above)
Comment (Ruslan357) [reply 3.14.py:96]: Done!

File: 3.14.py (hunk 1 above)
Comment (Ruslan357) [reply 3.14.py]: then the last character is not obtained № 70 (

**Comment by auraz** [comment]
Please create .gitignore file in the main folder (read about it) and put .idea in it, and commit.

**Review by auraz — APPROVED** [review]
Wonderful! Good job!

File: 3.14.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
+                #Exercise 1 
+import swampy
+
+def right_justify(s):
+    s = ' '*69 + s
```
Comment (auraz) [inline 3.14.py]: Add len back

File: 3.14.py (hunk 1 above)
Comment (Ruslan357) [reply 3.14.py]: Done!

**Review by auraz — APPROVED** [review]

**Pull request merged**